test/testjsonld-cpp/CMakeLists.txt

```diff
@@ -1,66 +1,123 @@
-add_executable(UnitTests_jsonld-cpp main.cpp test_JsonLdApi.cpp test_JsonLdUtils.cpp test_FileLoader.cpp testHelpers.cpp testHelpers.h test_NodeComparisons.cpp test_ObjectComparisons.cpp test_UniqueNamer.cpp test_DoubleFormatter.cpp test_Permutator.cpp test_NormalizeUtils.cpp test_Sha1.cpp test_Regex.cpp test_RDFDataset.cpp ManifestLoader.cpp ManifestLoader.h UrdnaManifestLoader.cpp UrdnaManifestLoader.h TestCase.cpp TestCase.h UriBaseRewriter.cpp UriBaseRewriter.h TestCaseOptions.cpp TestCaseOptions.h test_RDFDatasetUtils.cpp test_Uri.cpp test_UriParser.cpp test_Sha256.cpp)
-
-target_compile_features(UnitTests_jsonld-cpp PRIVATE cxx_std_11)
-target_compile_options(UnitTests_jsonld-cpp PRIVATE ${DCD_CXX_FLAGS})
-set_target_properties(UnitTests_jsonld-cpp PROPERTIES CXX_EXTENSIONS OFF)
-
-target_include_directories(UnitTests_jsonld-cpp
-        PUBLIC
-        ${PROJECT_SOURCE_DIR}/libjsonld-cpp)
-
-target_link_libraries(UnitTests_jsonld-cpp jsonld-cpp Boost::filesystem gtest gmock rapidcheck_gtest)
-
+#
+# Build common testfiled into lib so they only need to be compiled once
+#
+add_library(
```

Comment:
thank you so much for making this library ... I knew that compiling all these damn files over and over again was annoying but I never got around to fixing the makefile

Reply:
Hopefully it will save a bit of time :)